lib/common/actions/index.js

```diff
@@ -132,9 +132,9 @@ function getErrorMessageFromJson (
     }
     // re-assign error message after it gets used in the detail.
     errorMessage = json.message || JSON.stringify(json)
+    // This section (L135 - L146) is a hack to display links to patterns referenced by a stop to delete.
```

Comment:
Can we add a TODO: REMOVE HACK above this line? Just to make it clear this has to go

Reply:
Nothing I would love more: https://github.com/ibi-group/datatools-ui/pull/943/commits/1203494986f3560476ac9251715238cb5c4a9fc1